system/Pager/PagerRenderer.php

```diff
@@ -82,6 +82,21 @@ class PagerRenderer
      */
     protected $pageSelector;
 
+    /**
+     * Returns the number of results per page that should be shown.
+     */
+    protected ?int $perPage;
+
+    /**
+     * The number of items the page starts with.
+     */
+    protected ?int $perPageStart;
+
+    /**
+     * The number of items the page ends with.
+     */
+    protected ?int $perPageEnd;
```

Comment:
I prefer to have null default value for these properties on the first place, to avoid unitilzed value, since it filled by `if()` check on `updatePerPages` which rely on another property flag

Reply:
Done!